swagger.yaml

```diff
@@ -1,22 +1,25 @@
 swagger: '2.0'
 info:
   title: Analytics Core API
-  version: '0.1'
+  version: '0.2'
 basePath: /api/v1/
+host: "f8a-analytics-2445582058137.production.gw.apicast.io"
 consumes:
   - application/json
 produces:
   - application/json
+
+securityDefinitions:
+  tokenAuth:
+    type: apiKey
+    name: user_key
+    in: query
+    description: Token Auth is used for Authentication
+
+security:
+  - tokenAuth: []
+
 paths:
-  /liveness:
```

Comment:
Is there a reason for removing liveness end point from spec. I agree, it is not useful from the documentation perspective.

Reply:
@arajkumar Liveness is removed because it is not defined in 3Scale, So, whenever user tries Interactive Mode, he will not be able to get 200 from this endpoint